src/server/scripts/Spells/spell_mage.cpp

```diff
@@ -1364,7 +1366,7 @@ void AddSC_mage_spell_scripts()
     new spell_mage_conjure_refreshment();
     new spell_mage_fire_frost_ward();
     new spell_mage_focus_magic();
-    new spell_mage_frostbolt();
+    new spell_mage_trigger_chilled();
```

Comment:
ordering in file (look, it actually is sorted alphabetically, who would have expected that)